src/lbaf/Utils/functions.py

```diff
@@ -0,0 +1,17 @@
+"""Utility functions"""
+import os
+from typing import Union
+
+def abspath_from(path: Union[str, None], relative_to_dir: str) -> Union[str, None]:
+    """Get the absolute path from the given path relative to specific directory.
```

Comment:
Add space here after triple double quotes

Reply:
@ppebay are you sure ? in doctring python documentations I often see there is no space like there: https://peps.python.org/pep-0257/